nemo_text_processing/text_normalization/en/graph_utils.py

```diff
@@ -79,20 +79,34 @@
 delete_extra_space = pynini.cross(pynini.closure(NEMO_WHITE_SPACE, 1), " ")
 delete_preserve_order = pynini.closure(
     pynutil.delete(" preserve_order: true")
-    | (pynutil.delete(" field_order: \"") + NEMO_NOT_QUOTE + pynutil.delete("\""))
+    | (pynutil.delete(' field_order: "') + NEMO_NOT_QUOTE + pynutil.delete('"'))
 )
 
+# String literal insertions
```

Comment:
Hmm, do we have no where else to put these? I guess they need to accessible to both taggers and classifiers, but putting them in the graph_utils seems odd.

Reply:
Considering that instead of operations on the strings, we just want simple pointers AND that `graph_utils.py` already has a bunch of pointers to strings, I think it's as good a place as any other.  It may also make it easier for imports.